Platformio/OmoteUI/wifiSettings.cpp

```diff
@@ -52,7 +52,7 @@ void OmoteUI::password_field_event_cb(lv_event_t* e)
   const char* password = lv_textarea_get_text(ta);
   switch(code){
     case LV_EVENT_READY:
-      this->mHardware->wifi_connect.notify(std::make_shared<std::string>(std::string(ssid)), std::make_shared<std::string>(std::string(password)));
```

Comment:
this->
is only needed if you have a local variable that shadows a member variable in order to make it unambiguous. 

Every variable declared in the header is available in any non-static member-function of the class no need to specify this->  

we can leave them if you find it helpful for readability?

Reply:
I know that it is possible to not use the "this->", however in my opinion it is more clear which variable I want and if someone later adds a shadowed variable it would still not break.
It would also be possible that someone might create a global static variable within the file which might shadow the object field.